README.md

```diff
@@ -1 +1,113 @@
-# terraform-cloud-connect
\ No newline at end of file
+# terraform-cloud-connect
+
+## Description
+
+The main purpose of this module is to create an IAM User with enough permission and a Security Group with correct Inbound and Outbound rules so that Bitmovin's Cloud Connect product will be able to run Video Encodings in the configured infrastructure.
+
+The Terraform Cloud Connect module does the following:
+
+- Creates new IAM user
+- Assigns required permission required for Cloud-Connect: ~ AmazonEC2FullAccess
+- Creates a Security Group
+- Adds required Inbound and Outbound rules
+- Outputs:
+    - Account Id
+    - Access Key
+    - Secret Access Key
+    - Security Group Id
+
+The above outputs is required for the configuration of Bitmovin's Cloud Connect.
+
+## Usage
+
+Install Terraform: 
+
+https://developer.hashicorp.com/terraform/install
+
+Run:
+
+````
+terraform init
+terraform plan
+terraform apply
+````
+
+You can use the Bitmovin Cloud Connect Terraform module like this:
+
+````
+module "bitmovin_cloud_connect" {
+  source  = "bitmovin/terraform-cloud-connect"
+
+  user_name = "bitmovin-cloud-connect-user"
+  policy_name = "bitmovin-inline-policy"
+  security_group_name = "bitmovin-security-group"
+
+  live_rtmp = false
+  live_srt = false
+  live_zixi = false
+
+  live_ingress_cidr_blocks = ["0.0.0.0/0"]
+  live_ingress_ipv6_cidr_blocks = ["::/0"]
+
+  tags = {
+           company = "bitmovin",
+           product = "cloud-connect"
+         }
+}
+````
+
+We need the following output:
+
+````
+output "account_id" {
+  value = module.bitmovin_cloud_connect.account_id
+}
+
+output "access_key" {
+  value = module.bitmovin_cloud_connect.access_key
+}
+
+output "secret_access_key" {
+  value     = module.bitmovin_cloud_connect.secret_access_key
+  sensitive = true
+}
+
+output "security_group_id" {
+  value = module.bitmovin_cloud_connect.security_group_id
+}
+````
+
+Print out the outputs:
+
+````
+terraform output -json
+````
+
+Remove the created resources with the following commands:
+
+````
+terraform destroy
+````
+
+**Warning**: using the above command will remove the user, policy, security group and rules needed for the Bitmovin Cloud Connect product to work. Use the destroy command when you want to stop using the previously created resources.
+
+## Inputs
```

Comment:
***Suggestion*** There is an inputs section but no output section. From a structural point that would be nice to sort input/outputs close to each other.

Reply:
I forgot about this. 

I've checked it now. It is hard to find what to write on the outputs:

We get: account_id, access_key, secret_access_key, security_group_id

All are strings and all are needed so we can control the newly created account from our solution.

Also,  secret_access_key is kind of sensitive information, but we cannot take control without it.

I've wrote this Readme.md based on 3 other Readme.md from our public repositories and 2 examples Terraform repositories. I took this Input chapter idea from one of them.

I am not sure I have what to write for the Outputs so I would skip it, although I agree with you that structural it makes sense.